client/src/pages/profile/BasicInfoInput/ArrayAttribute/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, FC } from 'react';
+import { Dropdown, Input } from 'semantic-ui-react';
+
+import { IArrayAttribute } from './types';
+
+const ArrayAttribute: FC<IArrayAttribute> = ({
+  label,
+  value,
+  options,
+  onChange,
+  onAddItem,
+  viewable,
+  editable,
+}): ReactElement => (
+  <>
+    {viewable && (
+      <div className="input-field">
+        <span>{`${label}:`}</span>
+        {editable ? (
+          <Dropdown
+            className="dropdown-field"
+            options={options}
+            search
+            selection
+            floating
+            multiple
+            allowAdditions
+            value={value}
+            onAddItem={onAddItem}
+            onChange={onChange}
+          />
+        ) : (
+          <Input
+            value={value
+              .map(
+                (el) => el.charAt(0).toUpperCase() + el.slice(1).toLowerCase(),
+              )
+              .join(', ')}
+            transparent
```

Comment:
The reason I didn't do disabled is because that really darkens the CSS and it makes it hard to read what was already there, lemme know if you still think it should be disabled

Reply:
Oh, I see. Yeah, it turns out readOnly does do something so maybe you could use that? It won't affect the text but the blinking editing cursor won't show up.

Reply:
I'll look into it 👍